libs/payments/stripe/src/lib/stripe.client.spec.ts

```diff
@@ -25,9 +44,9 @@ describe('StripeClient', () => {
     it('returns an existing customer from Stripe', async () => {
       const mockCustomer = CustomerFactory();
 
-      mockClient.stripe.customers.retrieve = jest
-        .fn()
-        .mockResolvedValueOnce(mockCustomer);
+      (mockClient.stripe.customers.retrieve as jest.Mock).mockResolvedValueOnce(
+        mockCustomer
+      );
```

Comment:
If you use

```
jest.spyOn(mockClient.stripe.customers, 'retrieve').mockResolvedValueOnce(
  mockCustomer,
);
```

You won't have to cast and I believe it should work just the same.